tests/test.sh

```diff
@@ -31,12 +31,28 @@ test_bin() {
   local expect=$1; shift
   local RESULT=-1
   cd $name;
-  bundle exec metadata-json-lint $* metadata.json >/dev/null 2>&1
+  bundle exec metadata-json-lint $* metadata.json >last_output 2>&1
   RESULT=$?
   if [ $RESULT -ne $expect ]; then
-    fail "Failing Test '${name}' (bin)"
+    fail "Failing Test (unexpected exit code) '${name}' (bin)"
   else
-    echo "Successful Test '${name}' (bin)"
+    # If the test is not expected to succeed then it should match an expected output
+    if [ $expect -eq $SUCCESS ]; then
+      echo "Successful Test '${name}' (bin)"
+    else
+      if [ -f expected ]; then
+        if grep --quiet "`cat expected`" last_output; then
+          echo "Successful Test '${name}' (bin)"
+        else
+          fail "Failing Test (did not get expected output) '${name}' (bin)"
```

Comment:
Same here - I realize the file contents are catted out but it's nice to show the filename if the person needs to review or add details in an issue.